fe/fe-core/src/main/java/org/apache/doris/datasource/hive/HMSExternalTable.java

```diff
@@ -1074,6 +1075,20 @@ public boolean isPartitionedTable() {
     public void beforeMTMVRefresh(MTMV mtmv) throws DdlException {
     }
 
+
+    public boolean firstColumnIsString() {
+        List<Column> columns = getColumns();
+        if (columns == null || columns.isEmpty()) {
+            return false;
+        }
+        return columns.get(0).getType().isScalarType(PrimitiveType.STRING);
+    }
+
+    public boolean canReadHiveJsonInOneColumn() {
+        return ConnectContext.get().getSessionVariable().isReadHiveJsonInOneColumn()
```

Comment:
There is a `sessionVariable` instance in `HiveScanNode`. Use it instead of `ConnectContext.get().getSessionVariable()`